version/version.go

```diff
@@ -0,0 +1,30 @@
+package version
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// This module provides the ability to inject Build (git sha) and Version information at compile time.
+// To set these values invoke go build as follows
+// go build -ldflags “-X github.com/lyft/flytepropeller/version.Build=xyz -X github.com/lyft/flytepropeller/version.Version=1.2.3"
+// NOTE: If the version is set and server.StartProfilingServerWithDefaultHandlers are initialized then, `/version`
```

Comment:
I don't think this comment is correct... it's only correct if you use the stdlib's version of these vars :)

Reply:
I used the same package. And I missed it